cmake/KokkosConfig.cmake.in

```diff
@@ -13,6 +13,7 @@ SET(Kokkos_LIBRARIES       @KOKKOS_EXT_LIBRARIES@)
 SET(Kokkos_DEVICES                      @KOKKOS_ENABLED_DEVICES@)
 SET(Kokkos_OPTIONS                      @KOKKOS_ENABLED_OPTIONS@)
 SET(Kokkos_ARCH                         @KOKKOS_ENABLED_ARCH_LIST@)
+SET(Kokkos_CXX_STANDARD                 @KOKKOS_CXX_STANDARD@)
```

Comment:
Why did you add this?

Reply:
I had intentions for doing user-friendly errors check downstream when Kokkos loads that needed this variable. I will remove this for now - but this may be a useful variable to export.

Reply:
Drop that line and you've got my approval

Reply:
Removed. Squashed onto previous commit.